src/main.js

```diff
@@ -74,8 +81,8 @@ const examples = [
       'future',
     directory: 'bus-stop-timetable',
     endpoint: {
-      path: 'bus/stop/.../timetable',
-      docsUrl: LEGACY_DOCS_URL
+      path: 'bus/stops',
```

Comment:
If we are changing this we should update the example in `src/examples/bus-stop-timetable/main.js` and probably  `response.json`, because it still uses `https://transportapi.com/v3/uk/bus/stop/450024834/timetable.json?app_id=${appId}&app_key=${appKey}`

Reply:
My mistake. The path should be `bus/stop_timetables`. I fixed the request path in the example. The response is the same.